extensions/reviewed/MultitouchJoystick.json

```diff
@@ -2511,45 +2784,30 @@
           ],
           "parameters": [
             {
-              "codeOnly": false,
-              "defaultValue": "",
               "description": "Object",
-              "longDescription": "",
               "name": "Object",
-              "optional": false,
-              "supplementaryInformation": "",
               "type": "object"
             },
             {
-              "codeOnly": false,
-              "defaultValue": "",
               "description": "Behavior",
-              "longDescription": "",
               "name": "Behavior",
-              "optional": false,
               "supplementaryInformation": "MultitouchJoystick::MultitouchJoystick",
               "type": "behavior"
             },
             {
-              "codeOnly": false,
-              "defaultValue": "",
               "description": "Value",
-              "longDescription": "",
               "name": "Value",
-              "optional": false,
-              "supplementaryInformation": "",
               "type": "yesorno"
             }
           ],
           "objectGroups": []
         },
         {
-          "description": "Simulate a touch on joystick (based on position).  Can be used for mouse controls.",
+          "description": "Simulate a touch on joystick (based on position).  Can be used for mouse controls. This action is deprecated.",
           "fullName": "Simulate a touch on joystick (based on position)",
           "functionType": "Action",
-          "group": "",
           "name": "SimulateTouch_Position",
-          "private": false,
+          "private": true,
```

Comment:
I hide these actions.
What should they be used for?
I don't understand what "mouse controls" have to do with touch control. Games should hide the virtual joystick on a computer or if a gamepad is connected to the phone so I don't see a usage for it.

Reply:
My original inspiration was from the platformer behavior that has "Simulate jump key pressed" actions.  
However, I found these actions very helpful when testing touch controls from a PC.

My preference is to leave these actions in the extension (not hidden, not deprecated), but this is not critical to me.

Reply:
If this is just for testing. They must be unhidden only during tests to avoid to confuse users.

Reply:
It is useful for testing the game controls when you are making a game on PC for a mobile device.

Reply:
I doubt it's useful to test it this way because it only tests something like 4 events where actions are already shared with keyboard events. What is important to test is playability and it can only be done with the real device.